packages/augur-ui/src/modules/market/containers/market-header-reporting.ts

```diff
@@ -8,24 +8,24 @@ import { updateModal } from 'modules/modal/actions/update-modal';
 import {
   MODAL_CLAIM_MARKETS_PROCEEDS,
   DESIGNATED_REPORTER_SELF,
+  MODAL_REPORTING,
 } from 'modules/common/constants';
 import { NodeStyleCallback } from 'modules/types';
 import { createBigNumber } from '../../../utils/create-big-number';
```

Comment:
might as well fix this reference to create-big-number to use 'modules/....' reference